### ISSUE TITLE
Vim keybindings?

### ISSUE THREAD

**Issue by MoS2DL** [plnkr/feedback#303]
I can't find any settings for using **vim keybindings**.

Are **vim keybindings** on the way?

**Comment by roblevintennis** [comment]
yes, very frustrating as a new user coming over from codepen where I use this all the time :(
